src/backend/distributed/commands/text_search.c

```diff
@@ -710,16 +710,19 @@ AlterTextSearchConfigurationSchemaStmtObjectAddress(Node *node, bool missing_ok)
 
 		if (!missing_ok && !OidIsValid(objid))
 		{
-			/*
-			 * if the text search config id is still invalid we couldn't find it, error
-			 * with the same message postgres would error with if missing_ok is false
-			 * (not ok to miss)
-			 */
-
-			ereport(ERROR,
-					(errcode(ERRCODE_UNDEFINED_OBJECT),
-					 errmsg("text search configuration \"%s\" does not exist",
-							NameListToString(castNode(List, stmt->object)))));
+			if (EnablePropagationWarnings)
```

Comment:
Again, we should try to avoid masking an error, this might be a Citus bug. 

Can this function return an InvalidObjectAddr, and let the caller return early and PG to give the proper error?

Reply:
It is left from the beginning of the PR. It is unnecessary again.